tasks/coverage/snapshots/estree_typescript.snap

```diff
@@ -2,7 +2,7 @@ commit: 15392346
 
 estree_typescript Summary:
 AST Parsed     : 10619/10725 (99.01%)
-Positive Passed: 9038/10725 (84.27%)
+Positive Passed: 8996/10725 (83.88%)
```

Comment:
Seems in correct.

Reply:
yeah saw this as well, see my comment here > https://github.com/oxc-project/oxc/issues/10465#issuecomment-2812943629

think the parser was already correct